src/components/PasswordInputComponent.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <label v-if="!props.repeat" class="title">Пароль</label>
+    <label v-else class="title">Повторите пароль</label>
```

Comment:
I suppose, that password repeating should be in another component.
So it would be possible to use this component in both registration views

Reply:
I think that it's a default functionality of password to be repeated and if it would be a new component, then there would be lots of repetitive code in project
I could suggest adding a second `div` with it's own logic for password if it's repeated, so in parent component you would write `<PasswordInput withRepeat />` instead of `<PasswordInput /><PasswordInput repeat />`, that way it would also be a separated logic of watching main password input and a repeated one. It's possible to do because I can't imagine a scenario of using password repeat without password input

Reply:
I agree with idea to add second ```div```